packages/calling-component-bindings/src/hooks/usePropsFor.ts

```diff
@@ -119,6 +123,11 @@ export const getSelector = <Component extends (props: any) => JSX.Element | unde
 };
 
 const findSelector = (component: (props: any) => JSX.Element | undefined): any => {
+  /* @conditional-compile-remove(dialpad) */
+  if (component === _Dialpad) {
+    return emptySelector;
```

Comment:
Add comment why this returns the emptySelector
e.g. `// Dialpad only has handlers currently and doesn't require any props from the stateful layer so return the emptySelector`